py_mmd_tools/nc_to_mmd.py

```diff
@@ -1581,8 +1586,9 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
         mmd_yaml.pop('geographic_extent')
 
         # Get use_constraint data
+        print('BEFORE')
         self.metadata['use_constraint'] = self.get_license(mmd_yaml.pop('use_constraint'), ncin)
-
+        print('AFTER')
```

Comment:
```suggestion
```

Reply:
Removed - sorry, I forgot to take this out.